README.md

```diff
@@ -9,3 +10,13 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Proof:
+
+Let $G_{1}$ be any graph with size n and vertices $u_{1}$, $v_{1}$, and $G_{2}$ be any graph with size n - i, where i is any non-zero positive integer, and with vertices $u_{2}$, $v_{2}$,
+
+For some edge $E_{1}$ in $G_{1}$, there only exists a function $f: u_{1} \rightarrow v_{1}$ such that $(u_{1}, v_{1}) \in E_{1}$ and $(f(u_{1}), f(v_{1})) \neq (u_{2}, v_{2})$
```

Comment:
What if there's a disconnected node somewhere, and that's the only difference to the other graph?